aspnetcore/migration/22-to-30.md

```diff
@@ -643,4 +643,4 @@ If you're using features of `Newtonsoft.Json` that aren't supported in `System.T
 
 ## Opt in to runtime compilation
 
-In 3.0, runtime compilation is an opt-in scenario. To enable runtime compilation, see https://docs.microsoft.com/aspnet/core/mvc/views/view-compilation?view=aspnetcore-3.0#runtime-compilation.
+In 3.0, runtime compilation is an opt-in scenario. To enable runtime compilation, see <xref:mvc/views/view-compilation#runtime-compilation>.
```

Comment:
I added `view=aspnetcore-3.0` here in case the guide is being viewed in 2.2 "mode". I thought that without it, the reader would be sent to the 2.2 version of the linked page in that scenario and the information wouldn't show.

Reply:
Unfortunately that doesn't work. Next week the default version will be 3.0.

cc @scottaddie